.eslintrc

```diff
@@ -1,4 +1,7 @@
 {
+  "parserOptions": {
```

Comment:
In that case, I'll add a system test later, and remove all string interpolation operations within the scripts.